docs/source/admin.rst

```diff
@@ -29,6 +29,10 @@ including port if necessary and click the Save button!
 
 |Base URL configuration|
 
+A command line tool for URL configuration is also available::
+
+    ./manage.py baseurl public.tenant.kiwitcms.org
+
```

Comment:
All of this is not an URL but a domain name. The documentation uses bad terminology to refer to this configuration and you've used bad naming everywhere else. Needs update.

Now that there is a command to modify this prefer using the command, so move the command example above the manual instructions.


Also this entire section should actually go under https://kiwitcms.readthedocs.io/en/latest/installing_docker.html#initial-configuration-of-running-container

Note: the URL for the docs is also used in middleware which tells the when this isn't configured!